public/script.js

```diff
@@ -1860,7 +1860,7 @@ async function Generate(type, { automatic_trigger, force_name2, resolve, reject,
                 arrMes = arrMes.reverse();
                 arrMes.forEach(function (item, i, arr) {//For added anchors and others
 
-                    if (i === arrMes.length - 1 && $.trim(item).substr(0, (name1 + ":").length) != name1 + ":") {
+                    if (i === arrMes.length - 1 && !item.trim().startsWith(name1 + ":")) {
```

Comment:
jQuery's `trim()` is deprecated. `startsWith()` is more efficient and so much easier to read.